src/jquery.mockjax.js

```diff
@@ -269,6 +269,7 @@
 				url: mockHandler.proxy,
 				type: mockHandler.proxyType,
 				data: mockHandler.data,
+				async: false,
```

Comment:
I can understand it gets messy, but I don't think it is appropriate for the testing library to override the asynchronous nature of the source code just to reduce messiness. Ultimately, you're overriding the nature of the request (switching from async to sync). If the source specifies a sync ajax call this would be good, but otherwise I don't think we should flip that switch.